practice_app/pages/index.js

```diff
@@ -187,6 +192,30 @@ export default function Home({ context }) {
         }
     }
 
+    // This function returns the translated version of the input, to the language spoken in user's location
+    // current functionality : source_lang=english, target_lang=ip_dependent
+    const getTranslation = async () => {
```

Comment:
You should wrap the body of this function in a `try... catch` block because if an error happens it will crash the frontend

Reply:
Done.